test/mocks/handlers/tripReq.handler.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/prefer-default-export */
+import { rest } from 'msw';
+import baseURl from '../../../src/api/dummy';
+
+export const tripReqHandlers = [
+  rest.get('https://mocki.io/v1/0a16d481-3752-4cc6-b7f8-ca40f1b95a3d', (req, res, ctx) => {
```

Comment:
Add the real endpoint for our app. because this mocking services doesn't know if the requester is authenticated or add an authentication error scenario.